### DISCUSSION TITLE
Write and Read limit in free cluster

### DISCUSSION THREAD

**Discussion by Yaelahfelix** [mongodb/mongoid#5771] [Q&A]
Hello,

I am new to MongoDB. Does MongoDB have write and read limits? I've been searching for information, and I've only found details about the connection limit. Does this mean MongoDB doesn't have specific write and read limits, or is there something else to limiting the user fetching like bandwith or something??

**Comment by alexbevi** [comment]
Hey @Yaelahfelix, this isn't really a question specific to the Mongoid ODM and is likely more appropriate for our [Community Forums](https://www.mongodb.com/community/forums/).

At a high level though, the answer to your question may be documented either at [_MongoDB Limits and Thresholds_](https://www.mongodb.com/docs/manual/reference/limits/) or [_Atlas Service Limits_](https://www.mongodb.com/docs/atlas/reference/atlas-limits/).

Regarding read/write limits, the [WiredTiger Storage Engine](https://www.mongodb.com/docs/manual/core/wiredtiger/) manages concurrency internally via "tickets" so there is a "limit" of 128 of each, but these refer specifically to how many concurrent operations can be in flight before the storage engine queues operations. It is a limit, but is more of an internal control than anything else.

If you have any questions about MongoDB in general, feel free to ask in the community forums.
